src/bit-systems/loop-animation.ts

```diff
@@ -0,0 +1,105 @@
+import { addComponent, defineQuery, enterQuery, exitQuery, hasComponent, removeComponent } from "bitecs";
+import { AnimationAction, AnimationClip, AnimationMixer, LoopRepeat } from "three";
+import {
+  MixerAnimatable,
+  MixerAnimatableData,
+  LoopAnimation,
+  LoopAnimationData,
+  LoopAnimationInitialize,
+  LoopAnimationInitializeData,
+  Object3DTag
+} from "../bit-components";
+import { HubsWorld } from "../app";
+
+const loopAnimationInitializeQuery = defineQuery([LoopAnimationInitialize, MixerAnimatable, Object3DTag]);
+const loopAnimationInitializeEnterQuery = enterQuery(loopAnimationInitializeQuery);
+
+const loopAnimationQuery = defineQuery([LoopAnimation, MixerAnimatable, Object3DTag]);
+const loopAnimationExitQuery = exitQuery(loopAnimationQuery);
+
+// Question: Who should have AnimationMixer?
+//           MixerAnimatable component or Scene/App?
```

Comment:
I think it makes sense to have "on" a MixerAnimatable component . Then when loop-animation components are added we find the nearest ancestor that is MixerAnimatable and use its mixer.